static/app/views/alerts/list/rules/combinedAlertBadge.tsx

```diff
@@ -0,0 +1,65 @@
+import AlertBadge from 'sentry/components/badge/alertBadge';
+import {Tooltip} from 'sentry/components/tooltip';
+import {t, tct} from 'sentry/locale';
+import {MonitorType} from 'sentry/types/alerts';
+import {
+  ActivationStatus,
+  type CombinedMetricIssueAlerts,
+  IncidentStatus,
+} from 'sentry/views/alerts/types';
+import {isIssueAlert} from 'sentry/views/alerts/utils';
+
+interface Props {
+  rule: CombinedMetricIssueAlerts;
+}
+
+const IncidentStatusText: Record<IncidentStatus, string> = {
+  [IncidentStatus.CRITICAL]: t('Critical'),
+  [IncidentStatus.WARNING]: t('Warning'),
+  [IncidentStatus.CLOSED]: t('Resolved'),
+  [IncidentStatus.OPENED]: t('Resolved'),
```

Comment:
Is this right?

Reply:
I know it's copy paste but, seems weird?

Reply:
yea I saw that too and was confused but didn't want to change any functionality with this refactor PR. Judging from the backend this could be a bug.
https://github.com/getsentry/sentry/blob/5b00015693fdc5dada4f7916853e04bf560c2139/src/sentry/incidents/models/incident.py#L168


 I'll open another PR to correct the text and maybe tag alerts team on it or dan if they have more context.

Reply:
https://github.com/getsentry/sentry/pull/75456